src/apps/explorer/state.ts

```diff
@@ -1,17 +1,23 @@
 import { GlobalState, GLOBAL_INITIAL_STATE, globalRootReducer } from 'state'
 import { Erc20State, INITIAL_ERC20_STATE, reducer as erc20sReducer } from 'state/erc20'
+import { reducer as networkReducer } from 'state/network'
+
 import combineReducers from 'combine-reducers'
+import { Network } from 'types'
 
 export type ExplorerAppState = GlobalState & {
   erc20s: Erc20State
+  networkId: Network | null
 }
 
 export const INITIAL_STATE = (): ExplorerAppState => ({
   ...GLOBAL_INITIAL_STATE(),
   erc20s: INITIAL_ERC20_STATE,
+  networkId: null,
```

Comment:
Why not start with state == mainnet?

Reply:
Oh, sorry I read this after merging!
Good question!

I don't think to trigger calls or anything heavy, or re-rendering before a tiny and fast app warm up.
So it's null for a few milliseconds, then it should have a networkId

Reply:
It's alright for now. We can revisit later.